log2ram.conf

```diff
@@ -14,4 +14,24 @@ USE_RSYNC=false
 
 # If there are some errors with available RAM space, a system mail will be send
 # Change it to false and you will have only a log if there is no place on RAM anymore.
-MAIL=true
\ No newline at end of file
+MAIL=true
+
+# Compression algorythm either lzo or lz4 on a Zero lzo provides lower process load if you have CPU power to spare then maybe
+# lz4 will provide better results but if you check the spreadsheet in the tests folder lzo on a pi zero at least was superior in
+# performance lz4 may give better compression.
+# all /proc/crypto algs are supported but s/w vs h/w obviously much slower.
+# but check if exist also shortened list via cat /sys/block/zramX/comp_algorithm
+COMP_ALG_LOG=deflate
+COMP_ALG_SWAP=lz4
+# ZLTG flag means use ZRAM ramdisk for log2ram true=enable / false=disable
```

Comment:
"ZL2R flag" ?
can you please describe the acronym in the comments (or use a more explicit one)?

Reply:
ZramLog2Ram